### PR TITLE
Update pod.md

### PR DESCRIPTION
**Pull request by mattayes**
Remove statements about the future, clean up formatting.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7497)

### CODE REVIEW

File: docs/concepts/workloads/pods/pod.md [hunk 1]
```diff
@@ -165,14 +164,14 @@ Because pods represent running processes on nodes in the cluster, it is importan
...
-6. When the grace period expires, any processes still running in the Pod are killed with SIGKILL.
-7. The Kubelet will finish deleting the Pod on the API server by setting grace period 0 (immediate deletion). The Pod disappears from the API and is no longer visible from the client.
+1. The Pod in the API server is updated with the time beyond which the Pod is considered "dead" along with the grace period.
+1. Pod shows up as "Terminating" when listed in client commands
+1. (simultaneous with 3) When the Kubelet sees that a Pod has been marked as terminating because the time in 2 has been set, it begins the pod shutdown process.
+    * If the pod has defined a [preStop hook](/docs/concepts/containers/container-lifecycle-hooks/#hook-details), it is invoked inside of the pod. If the `preStop` hook is still running after the grace period expires, step 2 is then invoked with a small (2 second) extended grace period.
```
Comment (heckj) [inline docs/concepts/workloads/pods/pod.md]: I’m not sure if you intended a change in the output or not - Jekyll doesn’t mix ordered and unordered lists, so the end result is that this “appears” the same in the rendered output (see https://deploy-preview-7497--kubernetes-io-master-staging.netlify.com/docs/concepts/workloads/pods/pod/#termination-of-pods for the list)

Comment (mattayes) [reply docs/concepts/workloads/pods/pod.md]: @heckj Thanks for the heads up on Jekyll, I didn't know that. I'll change those `*`s for `1`s.

**Pull request merged**